crates/re_space_view_spatial/src/parts/images.rs

```diff
@@ -608,20 +608,28 @@ impl NamedViewSystem for ImagesPart {
 }
 
 impl ViewPartSystem for ImagesPart {
-    fn archetype(&self) -> ArchetypeDefinition {
-        // TODO(jleibs): Figure out the right thing to do here.
-        // What this actually wants to do is indicate that it is interested
-        // in all 3 indicator archetypes.
-        vec1::vec1![TensorData::name()]
+    fn required_components(&self) -> IntSet<ComponentName> {
+        Image::required_components()
+            .iter()
+            .chain(DepthImage::required_components().iter())
+            .chain(SegmentationImage::required_components().iter())
+            .map(ToOwned::to_owned)
+            .collect()
```

Comment:
I think this should just be `Image::required_components()`, or even more leniently, the *intersection* of the 3 required_components.

Suppose DepthImage and SegmentationImage *did* in fact add their own non-overlapping required_components. Then the reality is no Image would have all of the components we are looking for here.

Reply:
oof, yes, nice catch